docs/replication-retry.md

```diff
@@ -18,21 +18,47 @@ See [Explanations](#explanations) for more detail.
 
 ## Definition of API
 
-* GET `/_/crr/failed`
+* GET `/_/crr/failed?marker=marker`
 
     This GET request retrieves a listing of all failed operations. This
     operation is useful if you're interested in whether any CRR operations have
     failed and want the entire listing.
 
-    Response:
+    Response that is not truncated:
+
+    ```sh
+    {
+        IsTruncated: false,
+        Versions: [{
+            Bucket: <bucket>,
+            Key: <key>,
+            VersionId: <version-id>,
+            StorageClass: <site>,
+            Size: <size>,
+            LastModified: <last-modified>,
+        }],
+        MaxKeys: 100,
```

Comment:
Is 100 the default value? I would recommend to use the same default than S3 default (1000 AFAIK) for the principle of least surprise and also because it may help with large listings.